src/screens/SignIn/SignIn.tsx

```diff
@@ -0,0 +1,225 @@
+/* eslint-disable react-native-a11y/has-valid-accessibility-descriptors */
```

Comment:
Remove this line

Reply:
Removed and added necessary props to input fields. Also made the password field hide the text written to it.